clients/fides-js/src/fides-tcf.ts

```diff
@@ -0,0 +1,131 @@
+/**
```

Comment:
Refactored the original `fides.ts` so that much of its functional logic is in the new `initialize.ts` file. Now `fides-tcf` and `fides.ts` both use those functions